package.json

```diff
@@ -4,6 +4,9 @@
   "license": "MIT",
   "scripts": {
     "dev": "npx nx run-many --target=dev --projects=server,client",
+    "lint": "npx nx run-many --target=lint --projects=server,client",
+    "test": "npx nx run-many --target=test --projects=server,client",
+    "preinstall": "npm i nx",
```

Comment:
This caused problem on me. When I run `npm i`, both package.json and lock file has been changed. I dont know the reason, but if it is the preinstall, you can either solve this problem or remove preinstall.

Reply:
i guess i will remove